lib/rspecq/worker.rb

```diff
@@ -230,7 +273,7 @@ def reset_rspec_state!
     # falling back to scheduling them as whole files. Their errors will be
     # reported in the normal flow when they're eventually picked up by a worker.
     def files_to_example_ids(files)
-      cmd = "DISABLE_SPRING=1 bundle exec rspec --dry-run --format json #{files.join(' ')}"
+      cmd = "env -u COVERAGE DISABLE_SPRING=1 bundle exec rspec --dry-run --format json #{files.join(' ')}"
```

Comment:
can't we remove COVERAGE env in the top process instead of this?

something like:

```
env -u COVERAGE bundle exec rspecq ...
```

Reply:
wouldn't that make it that our pipeline doesn't upload the coverage when we run the actual tests later 🤔?

Reply:
ah, I see